### PR TITLE
add Chaikin Osc (A/D Osc) to ta menu

### PR DESCRIPTION
**Pull request by devwolff**

### CODE REVIEW

**Review by DidierRLopes — COMMENTED** [review]
Captura de ecrã 2021-08-21, às 12 37 03

Couple of suggestions:

- Price to not have $ as we can load data where dollar isn't the price unit
- Volume is quite big, i.e. seeing `1e8` on the label is a bit ugly. Could we have we rename column to Volume [M] and divide the units by 1e6? Same for AD Osc column

**Comment by devwolff** [comment]
@DidierRLopes It seems to be the issue for other volume plots too. I will fix them up

**Comment by devwolff** [comment]
Figure_1

**Comment by DidierRLopes** [comment]
> @DidierRLopes It seems to be the issue for other volume plots too. I will fix them up

Thank you, you a legend!

File: gamestonk_terminal/common/technical_analysis/volume_view.py [hunk 1]
```diff
@@ -140,42 +140,54 @@ def plot_adosc(
     else:
         bar_width = timedelta(minutes=int(s_interval.split("m")[0]))
 
+    divisor = 1000000
```
Comment (DidierRLopes) [inline gamestonk_terminal/common/technical_analysis/volume_view.py:141]: Not something to be done now.

Just for future, Python3 has this feature that allows you to add "_" in between to make a number more readable.

E.g. `1_000_000`

**Comment by DidierRLopes** [comment]
I'm still trying to figure out what's wrong with the tests, I'll merge this once I fix that :)

**Review by DidierRLopes — APPROVED** [review]

**Comment by DidierRLopes** [comment]
#694 does it

**Pull request merged**